app/Http/Controllers/BeatmapDiscussionPostsController.php

```diff
@@ -57,12 +57,12 @@ public function destroy($id)
      *   The response of this endpoint is likely to change soon!
      * </aside>
      *
-     * Field       | Type                                                    | Description
-     * ----------- | ------------------------------------------------------- | -----------
-     * beatmapsets | [BeatmapsetCompact](#beatmapsetcompact)                 | |
-     * cursor      | [Cursor](#cursor)                                       | |
-     * posts       | [BeatmapsetDiscussionPost](#beatmapsetdiscussionpost)[] | |
-     * users       | [UserCompact](#usercompact)                             | |
+     * Field         | Type                                                    | Description
+     * ------------- | ------------------------------------------------------- | -----------
+     * beatmapsets   | [BeatmapsetCompact](#beatmapsetcompact)                 | |
+     * cursor_string | string?                                                 | |
```

Comment:
I think this should reference what `cursor_string` is for instead of just being `string?`